.cirrus.yml

```
@@ -987,6 +987,22 @@ test_image_build_task:
     env:
         DRYRUN: 1
 
+fcos_image_build_task:
+    # Only a test. Does not push built images.
+    # Image push to quay will be handled by GHA in
+    # .github/workflows/fcos-podman-next-build.yml
+    <<: *image-build
+    name: "Test Build FCOS image $CTXDIR"
+    alias: fcos_image_build
+    only_if: $CIRRUS_PR != '' && $CIRRUS_BRANCH == 'main'
+    matrix:
+        - env:
+            CTXDIR: contrib/podman-next/fcos-podmanimage
+    env:
```

```suggestion
    env:
        CTXDIR: contrib/podman-next/fcos-podmanimage

```

Minor: No need for a matrix section here, and it could be confusing to maintainers.  Just stick it right into the normal `env` section.